### DISCUSSION TITLE
2.0-alpha1: openstack_auth_type vs os_auth_type

### DISCUSSION THREAD

**Discussion by sebastian-luna-valero** [tdviet/fedcloudclient#190] [Q&A] [answered]
Hi,

Are [openstack_auth_type](https://github.com/tdviet/fedcloudclient/blob/2.0-alpha1/fedcloudclient/conf.py#L22) and [os_auth_type](https://github.com/tdviet/fedcloudclient/blob/2.0-alpha1/fedcloudclient/conf.py#L38) redundant?

**Comment by tdviet** [comment, marked as answer]
Yes, it was leftover after renaming. Fixed in https://github.com/tdviet/fedcloudclient/commit/9c18eeb101c797130a82daeca2a8285eb536764e
